exercise_01/transformStringToCollection.test.js

```diff
@@ -0,0 +1,67 @@
+import transformStringToArray from "./transformStringToCollection.js";
+
+describe("Given function transformStringToArray", () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+  const mockOnError = jest.fn().mockImplementation(() => []);
+  const mockOnSuccess = jest
+    .fn()
+    .mockImplementation(() => ["Bon", "dia", "em", "dic", "Marc"]);
+
+  test("When the value type of is a number then empty array should be returned", () => {
+    //Arrange
+    const number = 12;
+    //Act
+    const result = transformStringToArray(number, mockOnError, mockOnSuccess);
+    //Assert
+    expect(result).toEqual([]);
+    expect(mockOnError).toHaveBeenCalledTimes(1);
+  });
```

Comment:
Improve Code Readability with Clearer Code Blocks. Fix it!
```suggestion
describe("Given function transformStringToArray", () => {
  beforeEach(() => {
    jest.clearAllMocks();
  });
  
  const mockOnError = jest.fn().mockImplementation(() => []);
  const mockOnSuccess = jest
    .fn()
    .mockImplementation(() => ["Bon", "dia", "em", "dic", "Marc"]);

  test("When the value type of is a number then empty array should be returned", () => {
    //Arrange
    const number = 12;
    
    //Act
    const result = transformStringToArray(number, mockOnError, mockOnSuccess);
    
    //Assert
    expect(result).toEqual([]);
    expect(mockOnError).toHaveBeenCalledTimes(1);
  });
```